src/resolvers/queries/ScheduledEventQuery.ts

```diff
@@ -1,10 +1,32 @@
-import { Arg, Ctx, ID, Query, Resolver } from 'type-graphql';
+import { Arg, Ctx, Field, ID, InputType, Query, Resolver } from 'type-graphql';
 
 import { ResolverContext } from '../../context';
+import { TzLessDateTime } from '../CustomScalars';
 import { ScheduledEvent } from '../types/ScheduledEvent';
 
+@InputType()
+export class ScheduledEventFilter {
+  @Field(() => TzLessDateTime, { nullable: true })
```

Comment:
I wonder why do we want time to be without timezone? Could you foresee any issues because of this?

Reply:
In our current situations TZ adds no value but introduces some complexity. When I was working on the calendar I tried changing my TZ a lot and it broke many things, instead of playing with the TZ it was just easier to get rid of it.